static/app/views/replays/detail/layout/chooseLayout.tsx

```diff
@@ -24,7 +24,7 @@ function ChooseLayout({}: Props) {
     <RelativeContainer>
       <DropdownControl
         label={getLayoutIcon(getParamValue())}
-        buttonProps={{size: 'sm'}}
```

Comment:
The `<IconPanel>` size was and is still set to `sm` above in this file. I'm going to leave it alone because there's no text label on that icon right now, so I feel like the more spacious icon is nice because there's no label on the button.